boards/arm/thingy53_nrf5340/thingy53_nrf5340_common.dts

```diff
@@ -89,9 +89,24 @@
 		mode-gpios  = <&gpio1 12 GPIO_ACTIVE_HIGH>;
 		pdn-gpios   = <&gpio1 10 GPIO_ACTIVE_HIGH>;
 		tx-en-gpios = <&gpio0 30 GPIO_ACTIVE_HIGH>;
+		gpio-fwd = <&gpio_fwd>;
```

Comment:
What is the reason to link fem node to nrf-gpio-forwarder?

Reply:
At first I thought that it would be easier to follow this way, but it's unnecessary and turns out to be confusing. I'll remove the reference to `gpio-fwd` from FEM bindings